### PR TITLE
enh(core): upgrade storybook and remove dependencies

### PR DESCRIPTION
**Pull request by kduret**
- upgrade storybook to 5.3 (configuration update)
- replace emotionjs by material-ui styling solution
- replace classnames by clsx (lightweight)
- remove useless dependencies

### CODE REVIEW

**Comment by Thebarda** [comment]
Do we need to use clsx instead of classnames inside Centreon Web, BAM and Auto Disco ?

**Comment by kduret** [comment]
> Do we need to use clsx instead of classnames inside Centreon Web, BAM and Auto Disco ?

I think so cause it's more lightweight than classnames and it's the solution used by material-ui
Furthermore, we don't use extra features provided by classnames

File: src/storyshots.test.js [hunk 1]
```diff
@@ -14,12 +14,19 @@ const beforeScreenshot = () => {
     }, 600),
   );
 };
+const getScreenshotOptions = () => {
```
Comment (bdauria) [inline src/storyshots.test.js]: Why is this change necessary?

Comment (kduret) [reply src/storyshots.test.js]: Notre necessary 
At the beginning, I Wanted to remove background but finally I let it.
Then i will remove this function
